pkgs/applications/networking/cluster/k3s/update-script.sh

```diff
@@ -3,19 +3,24 @@
 
 set -x -eu -o pipefail
 
+MINOR_VERSION="${1:?Must provide a minor version number, like '26', as the only argument}"
+
 WORKDIR=$(mktemp -d)
-trap "rm -rf ${WORKDIR}" EXIT
+# trap "rm -rf ${WORKDIR}" EXIT
```

Comment:
```suggestion
trap "rm -rf ${WORKDIR}" EXIT
```

Reply:
Fixed! Forgot to uncomment that after debugging something